File: CS50_Works/2-Problem_sets/c-Week6_Problem_Sets-c/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

Height = get_int("Height: ")

while Height > 8 or Height <= 0:
    Height = get_int("Height: ")
spaces = Height - 1
hashes = 1


for i in range(Height):
    for j in range(spaces):
        print(' ', end="")
    for k in range(hashes):
        print('#', end="")
    hashes = hashes + 1
    spaces = spaces - 1
    print('')

File: CS50_Works/2-Problem_sets/c-Week1_Problem_Sets-c/mario-less/mario.c
#include <stdio.h>
#include "cs50.h"
#include <math.h>

int main(void)

{
    int j;
    do
    {
        j = get_int(" Pryamid Height ?");
        // getting user prymid height//
    }
    while (j < 1 || j > 8);

    if (j >= 1 && j <= 8)

        for (int i = 0 ; i < j ; i++)
        {

            for (int l = j - 1 ; l > i ; l--)

            {

                printf(" ");

            }

            for (int k = -1 ; k < i ; k++)
            {

                printf("#");

            }

            printf("\n");

        }

}